### ISSUE TITLE
scales ordering in precomputed info file 

### ISSUE THREAD

**Issue by d-v-b** [google/neuroglancer#237]
It looks like the order of the scale metadata has an effect on how neuroglancer renders multiscale data.

Somehow tensorstore seems to occasionally mis-order the entries in the `scales` field of `info` (possibly a side-effect of delayed evaluation?), which on its own should be OK, since the elements of `scales` can and should be sorted on their resolution, not on their index. But it seems that when this mis-ordering occurs, neuroglancer fails to render all the scales. e.g., when my lowest resolution scale is the first entry in `scales`, neuroglancer only detects that scale level and will not render higher resolution scales. A solution would be to ignore the order of the `scales` metadata element and instead generate the list of scales by parsing the `resolution` property of every element in `scales`.

**Comment by jbms** [comment]
Yeah, Neuroglancer should sort the scales, but I'd also be interested if you can share your tensorstore-using code that sometimes results in out-of-order scales.  Are you adding multiple scales concurrently?

**Comment by d-v-b** [comment]
No, I'm not adding them concurrently -- see https://github.com/google/tensorstore/issues/10 for a code example.
